<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OPA_MainWidget</class>
 <widget class="QWidget" name="OPA_MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Параметры отправки</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Праметры порта</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editPortParams">
          <property name="text">
           <string>-portname=COM1 -baudrate=19200 -databits=8 -parity=0 -stopbits=2 -flowcontrol=0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bnPortParams">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Интервал отправки</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinInterval">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="suffix">
           <string> msec.</string>
          </property>
          <property name="maximum">
           <number>100000000</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bnStartStop">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: palegreen;</string>
          </property>
          <property name="text">
           <string>Старт</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Данные</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="bnEditData">
          <property name="text">
           <string>Изменить</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bnSendReset">
          <property name="text">
           <string>Отправить&quot;Сброс&quot; 0x77</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="tab0x19">
         <attribute name="title">
          <string>Неисправности (Тип 0x19)</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QListWidget" name="listType0x19"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab0x03">
         <attribute name="title">
          <string>Уровень пожарной опсности (Тип 0x03)</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QScrollArea" name="scrollArea_0x03">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>863</width>
               <height>355</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="vlayDirections"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab0x14">
         <attribute name="title">
          <string>Положение дверей и крышек (Тип 0x4)</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QListWidget" name="listType0x14"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab0x02">
         <attribute name="title">
          <string>Пожарная сигнализация (Тип 0x02)</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QScrollArea" name="scrollArea_0x02">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>863</width>
               <height>500</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QSplitter" name="splitter">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <widget class="QTabWidget" name="tabCurrent">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="tabPosition">
                  <enum>QTabWidget::North</enum>
                 </property>
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="tab_3">
                  <attribute name="title">
                   <string>Сигналы к отправке</string>
                  </attribute>
                  <layout class="QVBoxLayout" name="verticalLayout_9">
                   <property name="spacing">
                    <number>2</number>
                   </property>
                   <property name="leftMargin">
                    <number>2</number>
                   </property>
                   <property name="topMargin">
                    <number>2</number>
                   </property>
                   <property name="rightMargin">
                    <number>2</number>
                   </property>
                   <property name="bottomMargin">
                    <number>2</number>
                   </property>
                   <item>
                    <widget class="QTableView" name="tableCurrent">
                     <property name="selectionBehavior">
                      <enum>QAbstractItemView::SelectRows</enum>
                     </property>
                     <property name="sortingEnabled">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
                <widget class="QFrame" name="frameButtons">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="bnRemoveAll">
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="../../configurator2/res.qrc">
                      <normaloff>:/signals/icons/one_down.ico</normaloff>:/signals/icons/one_down.ico</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="bnRemoveSelected">
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="../../configurator2/res.qrc">
                      <normaloff>:/signals/icons/all_down.ico</normaloff>:/signals/icons/all_down.ico</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::MinimumExpanding</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Поиск</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="editSearch">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_5">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::MinimumExpanding</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="bnAddSelected">
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="../../configurator2/res.qrc">
                      <normaloff>:/signals/icons/all_up.ico</normaloff>:/signals/icons/all_up.ico</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="bnAddAll">
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="../../configurator2/res.qrc">
                      <normaloff>:/signals/icons/one_up.ico</normaloff>:/signals/icons/one_up.ico</iconset>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QTabWidget" name="tabAvailable">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="tabPosition">
                  <enum>QTabWidget::South</enum>
                 </property>
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="tab">
                  <attribute name="title">
                   <string>Доступные сигналы</string>
                  </attribute>
                  <layout class="QVBoxLayout" name="verticalLayout_10">
                   <property name="spacing">
                    <number>2</number>
                   </property>
                   <property name="leftMargin">
                    <number>2</number>
                   </property>
                   <property name="topMargin">
                    <number>2</number>
                   </property>
                   <property name="rightMargin">
                    <number>2</number>
                   </property>
                   <property name="bottomMargin">
                    <number>2</number>
                   </property>
                   <item>
                    <widget class="QTableView" name="tableAvailable">
                     <property name="editTriggers">
                      <set>QAbstractItemView::NoEditTriggers</set>
                     </property>
                     <property name="selectionBehavior">
                      <enum>QAbstractItemView::SelectRows</enum>
                     </property>
                     <property name="sortingEnabled">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../configurator2/res.qrc"/>
 </resources>
 <connections/>
</ui>
